homeassistant/components/moon/sensor.py.rej

```diff
@@ -0,0 +1,23 @@
+--- homeassistant/components/moon/sensor.py
```

Comment:
This file doesn't seem correct.

Reply:
You are correct. Thank you for the catch. Commit pushed.